frontend/src/components/Table.tsx

```diff
@@ -1,61 +1,68 @@
-import { DataTable } from "primereact/datatable";
```

Comment:
Maybe rename the file unless you want to later refactor in the datatable item itself.

Reply:
Good point, should probably be separate files anyway